src/Resources/contao/languages/en/tl_inserttags.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of terminal42/contao-inserttags.
+ *
+ * (c) terminal42
+ *
+ * @license MIT
+ */
+
+// Fields
+$GLOBALS['TL_LANG']['tl_inserttags']['tag'] = ['Insert-Tag', 'Please enter your insert-tag.<br />You can use this tag in any content element using {{custom::your-tag}}'];
```

Comment:
Everywhere: It's `insert tag` in English, see https://docs.contao.org/manual/en/article-management/insert-tags/